modules/calendar/full_calendar/fullcalendar.css

```diff
@@ -0,0 +1,1413 @@
+/*!
```

Comment:
May not since this is a Module but a means to get the color changes to it will need to be done @nileshprasad137